kore/src/Kore/Builtin/Int/Int.hs

```diff
@@ -46,10 +45,12 @@ import Data.String (
 import Data.Text (
     Text,
  )
-import qualified Data.Text as Text
 import Kore.Internal.InternalInt
 import Kore.Internal.Pattern as Pattern
-import Kore.Internal.TermLike as TermLike
+import Kore.Internal.TermLike as TermLike hiding (
+    DomainValueF,
+    StringLiteralF,
```

Comment:
Why are these hidden now?

Reply:
They conflict with the external `Syntax.Pattern` ones